### PR TITLE
Hotfix/disable newrelic

### PR DESCRIPTION
**Pull request by pkfec**
## Summary (required)

Remove the NEW RELIC python agent from openFEC repo. 

- Resolves https://github.com/fecgov/fec-accounts/issues/201

_Include a summary of proposed changes._

1. Remove all new relic references from :
    - dev/stage/prod manifest files
    - requirements.txt
    - rest.py
2. Rename the NEW_RELIC_APP_NAME environment variable to APP-NAME
    - webservices/tasks/utils.py

## How to test the changes locally
- Deploy this hotfix branch to stage
- Monitor the api, celery-worker and kabana logs
- Test downloads/export functinoality 
- Test legal search, candidate or committee search, should get results back.

## Impacted areas of the application
List general components of the application that this PR will affect:
-  www.fec.gov

### CODE REVIEW

File: manifests/manifest_celery_worker_prod.yml [hunk 1]
```diff
@@ -5,10 +5,6 @@ stack: cflinuxfs3
 buildpack: python_buildpack
 env:
```
Comment (patphongs) [inline manifests/manifest_celery_worker_prod.yml]: @pkfec can you please add the APP_NAME variable here for prod?

Comment (pkfec) [reply manifests/manifest_celery_worker_prod.yml]: good catch @patphongs. I see the variable being added to the file, in my local. Not sure why its not showing up here.

Comment (pkfec) [reply manifests/manifest_celery_worker_prod.yml]: my bad, i forgot to save the file. thanks @patphongs . Please check now.

Comment (patphongs) [reply manifests/manifest_celery_worker_prod.yml]: All good, thanks!

**Pull request merged**